From: Baruch Siach <baruch@tkos.co.il>
Date: Tue, 8 Mar 2016 14:40:49 +0200
Subject: [PATCH] Fix: add missing sys/types.h header

The musl C library requires inclusion of sys/types.h for mode_t.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Patch status: sent upstream
(http://lists.lttng.org/pipermail/lttng-dev/2016-March/025594.html)

 src/common/runas.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/common/runas.h b/src/common/runas.h
index 2c5565af3646..ac1143eecf84 100644
--- a/src/common/runas.h
+++ b/src/common/runas.h
@@ -19,6 +19,7 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include <sys/types.h>
 #include <unistd.h>
 #include <pthread.h>
 
-- 
2.7.0

